src/lib/getCollectionBalanceOf.tsx

```diff
@@ -0,0 +1,40 @@
+import { COLLECTIONS } from "@/lib/consts"
+import { getPublicClient } from "@/lib/clients"
+import zora1155Abi from "@/lib/abi/zora-drop.json"
+import { Address } from "viem"
+
+const getCollectionBalanceOf = async (address: Address) => {
+  const balancesPromise = COLLECTIONS.map(async (collection) => {
```

Comment:
I think this outside wrapper should still be in your previous lib
- actual: you renamed the existing lib
- expected: break down the large lib into smaller, well-defined, libs